docs/3.x/tech/installation-guide/cleanspeak-search-engine.adoc

```diff
@@ -23,52 +23,52 @@ $ sudo rpm -i cleanspeak-search-engine-<version>.rpm
 
 == Debian
 
-Use the DEB bundle to install on a Debian system (including Ubuntu, Mint, and Debian base). Execute this command to install the CleanSpeak Search Engine DEB (replace with the correct version number):
+Use the DEB bundle to install on a Debian system (including Ubuntu, Mint, and Debian base). Execute this command to install the CleanSpeak Search Engine DEB (replace with the correct `<version>` number):
 
 [source,shell]
 ----
 $ sudo dpkg -i cleanspeak-search-engine-<version>.deb
 ----
 
-== Mac OS X
+== macOS
 
-To install on Mac OS, use the ZIP bundle. Extract the CleanSpeak Search Engine ZIP file anywhere on the file system. Remember where you extract the file. This location will be referred to as CLEANSPEAK_HOME. We suggest extracting this file to a directory like/usr/local/inversoft.
+To install on macOS, use the ZIP bundle. Extract the CleanSpeak Search Engine ZIP file anywhere on the file system. Remember where you extract the file. This location will be referred to as `CLEANSPEAK_HOME`. We suggest extracting this file to a directory like /usr/local/cleanspeak.
 
 [NOTE]
 ====
 You can install CleanSpeak in any directory including your home directory. This is a good choice for development machines or when you don't have or want to use sudo to install CleanSpeak.
 ====
 
-Once the zip bundle has been extracted, the directory structure should look similar. If you installed somewhere other than the default CLEANSPEAK_HOME, your directory structure would be different. This is only shown as an example.
+Once the zip bundle has been extracted, the directory structure should look similar. If you installed somewhere other than the default `CLEANSPEAK_HOME`, your directory structure would be different. This is only shown as an example.
 
 [source,shell]
 ----
-/usr/local/inversoft/config
-/usr/local/inversoft/config/keystore
-/usr/local/inversoft/config/cleanspeak.properties
-/usr/local/inversoft/cleanspeak-search-engine
+/usr/local/cleanspeak/config
+/usr/local/cleanspeak/config/keystore
+/usr/local/cleanspeak/config/cleanspeak.properties
+/usr/local/cleanspeak/cleanspeak-search-engine
```

Comment:
For each of these zip examples, did you download the zip and perform this step to ensure this is what the files look like?